src/App.js

```diff
@@ -1,10 +1,25 @@
 import { Global } from '@emotion/react';
+import { ROUTES } from 'constants/route';
+import Home from 'pages/Home/Home';
+import Signin from 'pages/Signin/Signin';
+import Signup from 'pages/Signup/Signup';
+import Todos from 'pages/Todos/Todos';
+import { Routes, Route, Navigate } from 'react-router-dom';
 import resetCss from 'styles/global';
 
-const App = () => (
-  <div>
-    <Global styles={resetCss}></Global>
-  </div>
-);
+const App = () => {
+  // checkToken : 토큰 유무를 확인 (default : true)
+  const checkToken = !!localStorage.getItem('token');
+  return (
+    <Global styles={resetCss}>
+      <Routes>
+        <Route path={ROUTES.HOME} element={checkToken ? <Navigate to="/todo" /> : <Home />}></Route>
```

Comment:
self closing 형태로 사용하면 좀 더 깔끔할거 같습니다 :)

그리고 Navigate에 to props에 전달하는 값에도 상수를 사용해주면 어떨까요?

```suggestion
        <Route path={ROUTES.HOME} element={checkToken ? <Navigate to={ROUTES.TODO} /> : <Home />} />
```

Reply:
앗.. 있어서는 안될 실수를.. 했네요^^; 
감사합니다 하준님 바로 반영하겠습니다!